Sources/Leaf/Data/LeafData.swift

```diff
@@ -24,9 +24,30 @@ public enum LeafData {
     case dictionary([String: LeafData])
     case array([LeafData])
     case future(Future<LeafData>)
+    case stream(AnyOutputStream<LeafData>)
     public typealias Lazy = () -> (LeafData)
     case lazy(Lazy)
     case null
+    
+    public static func codableStream<O : Async.OutputStream>(_ stream: O) -> LeafData where O.Output == Encodable {
```

Comment:
- [ ] why do we need both of these?
- [ ] comments
- [ ] i think just `.stream(_:)` would be a better name